app/assets/javascripts/map/services/PlaceService.js

```diff
@@ -55,7 +55,7 @@ define([
   'use strict';
 
   var urlDefaultsParams = {
-    baselayers: 'loss,forestgain,forest2000',
+    baselayers: 'loss,forestgain,forest2000,forest2010',
```

Comment:
Are you sure about this? I mean, it seems that you are setting by default both forest cover layers. Could you check it? Maybe I'm wrong...